CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# Contributing
+
+We're so excited you're interested in helping with SuperTokens! We are happy to help you get started, even if you don't have any previous open-source experience :blush:
+
+## New to Open Source?
+
+1. Take a look at [How to Contribute to an Open Source Project on GitHub](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github)
+2. Go through the [SuperTokens Code of Conduct](https://github.com/supertokens/supertokens-web-js/blob/master/CODE_OF_CONDUCT.md)
+
+## Where to ask Questions?
+
+1. Check our [Github Issues](https://github.com/supertokens/supertokens-web-js/issues) to see if someone has already answered your question.
+2. Join our community on [Discord](https://supertokens.com/discord) and feel free to ask us your questions
+
+### Prerequisites
+
+-   OS: Linux or macOS
+-   Nodejs & npm
+-   IDE: [VSCode](https://code.visualstudio.com/download)(recommended) or equivalent IDE
+
+### Project Setup
+
+1. Please setup `supertokens-core` by following [this guide](https://github.com/supertokens/supertokens-core/blob/master/CONTRIBUTING.md#development-setup). If you are not contributing to `supertokens-core`, please skip steps 1 & 4 under "Project Setup" section.
+2. `git clone https://github.com/supertokens/supertokens-web-js.git`, `supertokens-web-js` and `supertokens-root` should exist side by side within the same parent directory.
+3. `cd supertokens-web-js`
+4. Install the project dependencies
+    ```
+    npm run init
+    ```
+5. Add git pre-commit hooks
+    ```
+    npm run set-up-hooks
+    ```
+
+## Modifying Code
+
+1. Open the `supertokens-web-js` project in your IDE.
+2. You can start modifying the code.
+3. After modification, you need to build the project:
+    ```
+    npm run build-pretty
+    ```
+4. Make sure the linter passes:
+    ```
+    npm run lint
+    ```
+
+## Testing
+
+1. Navigate to the supertokens-root repository
+2. Start the testing environment
+
+    ```
+    ./startTestingEnv --wait
+    ```
+
+3. Open a new terminal in `supertokens-web-js` and run all tests
+
+    ```
+    npm run test
+    ```
```

Comment:
I mean.. this isn't accurate. The way we test this SDK is via auth-react SDK. So it should talk about setting auth-react SDK + core and then linking this SDK in auth react and then running tests for auth-react.

Reply:
So this is true to run the few unit tests (storage abstraction and querier). When we add the pipeline of testing with auth-react I will be mentioning the steps for that here too

Reply:
So then i can't merge this PR until then.